src/back/TicketManager/src/Core/TicketManager.Core.Domain/TicketManager.Core.Domain.csproj

```diff
@@ -7,4 +7,8 @@
         <WarningsAsErrors>Nullable</WarningsAsErrors>
     </PropertyGroup>
 
+    <ItemGroup>
+      <ProjectReference Include="..\TicketManager.Core.Contracts\TicketManager.Core.Contracts.csproj" />
+    </ItemGroup>
```

Comment:
The reason why `domain` and `contracts` are separate projects is to not allow that. Generally classes in `domain` and in `contracts` have different responsibilities and shouldn't be the same classes. Sometimes result of that is that we have to write same classes/enums twice.

Reply:
Yes, sure. TaxIdType enum got mixed up between those two projects